### ISSUE TITLE
Using the "ChartSeriesItem" nested tag in Chart throws and error in the console

### ISSUE THREAD

**Issue by PekoPPT** [telerik/kendo-vue#87]
## Current behavior

```
<chart :title-text="'Gross domestic product growth /GDP annual %/'" 
        :legend-position="'bottom'"
        :series-defaults-type="'area'"
        :category-axis-categories="categoryAxisData"
        :category-axis-major-grid-lines-visible="false"
        :category-axis-labels-rotation="'auto'"
        :value-axis-labels-format="'{0}%'"
        :value-axis-line-visible="false"
        :value-axis-axis-crossing-value="-10"
        :tooltip-visible="true"
        :tooltip-format="'{0}%'"
        :tooltip-template="tooltipTemplate"
        :theme="'sass'">
    <chart-series-item :name="'India'"
                        :data="indiaData"
                        :line-style="'smooth'">
    </chart-series-item>
    <chart-series-item :name="'World'"
                        :data="worldData"
                        :line-style="'smooth'">
    </chart-series-item>
    <chart-series-item :name="'Haiti'"
                        :data="haitiData"
                        :line-style="'smooth'">
    </chart-series-item>
</chart>
```
 
When using the ChartSeriesItem tags to define the data in the Chart component the following error appears in the console once the Vue project is started.
![image](https://user-images.githubusercontent.com/41293735/99277856-03620580-2837-11eb-9fca-b666d2f02da0.png)

## Expected behavior

```    
<chart-series-item :name="'India'"
                        :data="indiaData"
                        :line-style="'smooth'">
    </chart-series-item>
    <chart-series-item :name="'World'"
                        :data="worldData"
                        :line-style="'smooth'">
    </chart-series-item>
    <chart-series-item :name="'Haiti'"
                        :data="haitiData"
                        :line-style="'smooth'">
    </chart-series-item>
```

Using a definition like the above, we should be able to define Chart's data with no issues. 

## Minimal reproduction of the problem with instructions
1. Open [this project ](https://github.com/telerik/kendo-vue/files/5548101/chartdemo.zip)
1. In the console run "npm install"
1. In the console run "npm run serve"
1. Open the URL of the project and see the browser's console

**Comment by PekoPPT** [comment]
The workaround that can be applied for the current issue is to define the Chart as follows:

```
<template>
    <div>
        <chart :title-text="'Gross domestic product growth /GDP annual %/'"
                :series="series"
               :legend-position="'bottom'"
               :series-defaults-type="'area'"
               :category-axis-categories="categoryAxisData"
               :category-axis-major-grid-lines-visible="false"
               :category-axis-labels-rotation="'auto'"
               :value-axis-labels-format="'{0}%'"
               :value-axis-line-visible="false"
               :value-axis-axis-crossing-value="-10"
               :tooltip-visible="true"
               :tooltip-format="'{0}%'"
               :tooltip-template="tooltipTemplate"
               :theme="'sass'">
        </chart>
    </div>
</template>
```
Where the series variable is defined like this:
```
computed: {
    series () {
        return [
            { name:'India', data: this.indiaData, line: {style: 'smooth'}},
                { name:'World', data: this.worldData, line: {style: 'smooth'}},
                { name:'Haiti', data: this.haitiData, line: {style: 'smooth'}}
        ]
    } 
}
```

**Comment by esaramago** [comment]
I'm writing the error here, so anyone can find this solution by making a Google search:

`[Vue warn]: Error in mounted hook: "TypeError: Cannot read property '$options' of undefined"`

**Comment by zdravkov** [comment]
fixed in version 2021.2.617
